recipes/zol/meta.yaml

```diff
@@ -52,7 +52,8 @@ requirements:
     - slclust
     - trimal
     - gzip
-    - miniprot =0.7
+    - miniprot =0.7 # [x86_64]
+    - miniprot >=0.7 # [aarch64]
```

Comment:
miniprot introduced support for aarch64 in 0.13. 
I am not sure whether upgrading the x86_64 package will lead to problems.
If there are problems then they will be only in the aarch64 build that is brand new and this should be OKish.